cmd/plural/validation.go

```diff
@@ -192,3 +161,36 @@ func upstreamSynced(fn func(*cli.Context) error) func(*cli.Context) error {
 		return fn(c)
 	}
 }
+
+func (p *Plural) impersonate(fn func(*cli.Context) error) func(*cli.Context) error {
+	return func(c *cli.Context) error {
+		p.InitPluralClient()
+		path := manifest.ProjectManifestPath()
+		project, err := manifest.ReadProject(path)
+		conf := config.Read()
+		if err != nil {
```

Comment:
We probably shouldn't error here, or is it only used when the repo is already set up?

Reply:
yes, it's used when the repo is already set up